generators/python/core_utilities/sdk/query_encoder.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any
+
+try:
+    import pydantic.v1 as pydantic  # type: ignore
+except ImportError:
+    import pydantic  # type: ignore
```

Comment:
@armandobelardo should we update the import here

Reply:
Ah thank you! Yes definitely, made a new PR instead of merging main into this: https://github.com/fern-api/fern/pull/3629